force-app/main/default/classes/AccountHelper.cls

```diff
@@ -27,20 +27,65 @@ Remember:
 Always ensure that your helper methods are bulk-safe to handle multiple records efficiently. 
 */
 public with sharing class AccountHelper {
+     
+// When an account is inserted, change the account type to 'Prospect' if there is no value in the type field
+    public static void setTypeProspect(List<Account> accounts){
+        for (Account acc : accounts) {
+            if (acc.Type == null) {
+                acc.Type = 'Prospect';
+            }
+        }
+    }
+// When an account is inserted copy the shipping address to the billing address.
+    public static void addressCopy(List<Account> accounts){
+        for (Account acc : accounts) {
+            if (acc.ShippingStreet != null) {
+                acc.BillingStreet = acc.ShippingStreet;
+            }
     
-    public void setTypeProspect(SObject objects){
-
+            if (acc.ShippingCity != null) {
+                acc.BillingCity = acc.ShippingCity;
+            }
+    
+            if (acc.ShippingState != null) {
+                acc.BillingState = acc.ShippingState;
+            }
+    
+            if (acc.ShippingPostalCode != null) {
+                acc.BillingPostalCode = acc.ShippingPostalCode;
+            }
+    
+            if (acc.ShippingCountry != null) {
+                acc.BillingCountry = acc.ShippingCountry;
+            }
+        }        
+    }
+// When an account is inserted set the rating to 'Hot' if the Phone, Website, and Fax is not empty.
+    public static void setRating(List<Account> accounts) {
+        for (Account acc : accounts) {
+            if (acc.Phone != null && acc.Website != null && acc.Fax != null) {
+                acc.Rating = 'Hot';
+            }
+        }
+    }
+// * When an account is inserted create a contact related to the account with the following default values:
+// * LastName = 'DefaultContact'
+// * Email = 'default@email.com' 
+    public static void defaultContact (List<Account> accounts) {
+        List<Contact> contacts = new List<Contact>();   
+            for(Account acc : accounts){
+                Contact con = new Contact();
+                con.LastName = 'DefaultContact';
+                con.Email = 'default@email.com';
+                con.AccountId = acc.Id;
```

Comment:
Try declaring these variables at the top of the class so they are easy to change